WINDOWS_LINUX_SUBSYSTEM2.md

```diff
@@ -0,0 +1,257 @@
+
+:warning: We invite you to contact a teacher to do that operation together as it might be a bit scary. that way you can have a WhatsApp call / Facetime (choose the tool of your choice so the teacher can see your screen :man_mechanic:​) :warning:
+
+If we take my example (sorry it is in French) :
+
+![motherboard.jpg](images/wsl2_motherboard.jpg)
+
+So I googled "*mpg x570 plus enable virtualization*" and the first video in the results taught me how to do it. It is part of a developer life to search for information and apply them this way :nerd_face:.
+
+Follow the instructions you found to enable virtualization and come back here, we will be waiting :smile:.  
+
+
+# Setup instructions
+
+The following instructions will help you to get ready for [Le Wagon](http://www.lewagon.org) fullstack bootcamp:
+
+- Grab a text editor, where you'll spend your day and nights
+- Install a package manager
+- Pimp your Terminal
+- Setup git and GitHub
+- Install Ruby
+
+
+## Remote tools
+
+To be able to interact when you are not on site, we will be using two tools.
+
+### Zoom
+
+Zoom is a video conferencing, go to [zoom page](https://zoom.us/download) and under *Zoom Client for Meetings* hit the **Download** button. open the executable you have just downloaded, a progress bar will appear then zoom will start. Click on **Connexion** and create an account. Once connected, zoom should look like this:
+
+![zoom.jpg](images/zoom.jpg)
+
+You can close zoom now :wink:.
+
+### Teamviewer
+
+
+
+For the most extreme bugs, a teacher might have to take control over your machine. to be able to do that we will need the Teamviewer tool. Go to the [teamviewer download page](https://www.teamviewer.com/en/download). It should automatically detect your exploitation system, if it doesn't don't hesitate to chose it from the list at the top of the page. Click on **Download Teamviewer**. Open the executable you just have downloaded, leave default settings and click on **Accept**. Progress bar will appear then Teamviewer will start when the installation is over. it should look like this:
+
+![teamviewer.jpg](images/teamviewer.jpg)
+
+You can close Teamviewer now :wink:.
+
+:warning: We will not and cannot take control over your computer without your permission. Your teamviewer paswword changes everytime you restart the application :thumbsup: We will only use this tool in last resort if a teacher needs hands on your machine. :warning:
+
+## Windows Subsystem for Linux
+
+WSL is the development environment you are going to use, you can learn more about WSL [here](https://docs.microsoft.com/en-us/windows/wsl/faq).
+
+Click on **Start** and type **powershell**. Right click on **Windows Powershell (x86)** then click on **Run as administrator**. A blue box will appear, copy and paste the following command into that blue box:
+
+```powershellEnable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux```
```

Comment:
I think the markdown is broken from here on, you need to add a linebreak after the opening triple backtick and before the closing one 👌